.eslintrc

```diff
@@ -4,18 +4,27 @@
     "es6": true
   },
   "parser": "@typescript-eslint/parser", // Specifies the ESLint parser
-  "plugins": ["@typescript-eslint"],
+  "plugins": [
+    "@typescript-eslint"
+  ],
   "extends": [
     "eslint:recommended",
     "plugin:@typescript-eslint/eslint-recommended",
     "plugin:@typescript-eslint/recommended",
     "plugin:@typescript-eslint/recommended-requiring-type-checking",
-    "mongodb-js/node"
```

Comment:
fwiw: you can just swap `mongodb-js/node` to or add `mongodb-js/react` which uses that plugin.